api/restHandler/PipelineConfigRestHandler.go

```diff
@@ -3277,3 +3309,155 @@ func (handler PipelineConfigRestHandlerImpl) PipelineNameSuggestion(w http.Respo
 	}
 	writeJsonResp(w, err, suggestedName, http.StatusOK)
 }
+
+func validatejson(jsondoc pipeline.TemplateRequest, chartVersion string) bool {
+
+	schemaLoader := gojsonschema.NewReferenceLoader(fmt.Sprintf("file:///Users/aviralsrivastava/GolandProjects/devtron/tests/testdata/%s.json", chartVersion))
```

Comment:
this is hardcoded to your local path, this should be picked from the same place where chart was loaded